apps/edxapp/templates/services/flower/dc.yml.j2

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: DeploymentConfig
+metadata:
+  labels:
+    app: edxapp
+    service: flower
+    version: "{{ edxapp_flower_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+  name: "edxapp-flower-{{ deployment_stamp }}"
+  namespace: "{{ project_name }}"
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: edxapp
+        service: flower
+        version: "{{ edxapp_flower_image_tag }}"
+        deployment_stamp: "{{ deployment_stamp }}"
+        deploymentconfig: "edxapp-flower-{{ deployment_stamp }}"
+    spec:
+      # Prefer running pods on different nodes for redundancy
```

Comment:
Since we have only 1 replica, is it useful to specify a podAntiAffinity ?

Reply:
No this can be removed